FittedSheets/SheetContentViewController.swift

```diff
@@ -126,6 +139,45 @@ public class SheetContentViewController: UIViewController {
         self.contentWrapperView.layer.cornerRadius = self.treatPullBarAsClear ? 0 : self.cornerRadius
         self.childContainerView.layer.cornerRadius = self.treatPullBarAsClear ? self.cornerRadius : 0
     }
+
+    private var titleBarViewTopInset: CGFloat {
+        return (self.options.pullBarHeight + self.gripSize.height) / 2 + 2
+    }
+
+    private func setupTitleBarView() {
+        guard self.options.shouldExtendBackground == false else { return }
+
+        titleBarView.backgroundColor = self.titleBarBackgroundColor
+        titleLabel.attributedText = self.attributedTitle
```

Comment:
I don't think those do anything. Just keep repeating the pattern in case it actually does.